spa/src/utils/api.js

```diff
@@ -65,7 +65,7 @@ function put(endpoint, url, body, context, jwt) {
   }
 }
 
-function patch(endpoint, url, body, context, jwt) {
+async function patch(endpoint, url, body, context, jwt) {
```

Comment:
why this change? Such code is there for a year

Reply:
fixed